rigex-front-end/src/pages/Rig/Rig.js

```diff
@@ -1,10 +1,56 @@
-import React from 'react';
 import { useParams } from 'react-router-dom';
+import EditIcon from '@material-ui/icons/Edit';
+import AddIcon from '@material-ui/icons/Add';
 
-const Rig = () => {
+import './rig.scss';
+import rigImage from '../../assets/rig.svg'
+
+const Rig = ({listOfRigs, listOfWells}) => {
   const { id } = useParams();
 
-  return <h1>Rig Works!</h1>;
+  if (!listOfWells) {
+    listOfWells = []
+  }
+
+  var name = "Name of Rig"
+  listOfRigs.map(rig => {
+    if (rig.id === id) {
+      name = rig.name
+    }
+    return null
+  })
+
+  return (
+    <div className="wellPanel">
```

Comment:
This section of code is much more along the lines of what I like to see. Good job.

Reply:
That was my mistake - this was from Jeremy's branch so I could ensure his routing and props passed correctly from landing. I should have removed it from the commit

Reply:
Removed everything not needed for testing landing path routing and props